tests/src/FunctionalJavascript/ParagraphsPasteTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Drupal\Tests\paragraphs_paste\FunctionalJavascript;
+
+use Drupal\FunctionalJavascriptTests\WebDriverTestBase;
+use Drupal\Tests\paragraphs\FunctionalJavascript\LoginAdminTrait;
+
+/**
+ * Tests the creation of paragraphs by pasting random data.
+ *
+ * TODO: Move widget settings form tests in separate class.
+ *
+ * @group paragraphs_features
+ */
+class ParagraphsPasteTest extends WebDriverTestBase {
+
+  use LoginAdminTrait;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  protected static $modules = [
```

Comment:
Do we really have to add all those dependencies? I think in WebTests only paragraphs_paste_test should be needed.